indica/converters/line_of_sight.py

```diff
@@ -49,9 +49,45 @@ class LineOfSightTransform(CoordinateTransform):
     passes
         Number of passes across the plasma (e.g. interferometer
         with corner cube has passes=2)
-
+    beamlets
+        Number of beamlets in the LOS spot cross-section.
+        Currently only works with quadratic sequence (e.g.
+        1^2 = 1, 2^2 = 4, 3^2 = 9, 4^2 = 16, etc...)
+    spot_width
+        Width of the LOS spot in (m).
+    spot_shape
+        Shape of the spot. e.g. "round" or "square"
+    div_width
+        Divergence angle of the width of the spot in (radians).
+    focal_length
+        Focal length of the LOS in (m).
+
+    Examples:
+        Pinhole Cameras
+            origin - position vector of the detector
+            direction - direction vector from detector to the
+                pinhole
+            spot_shape - the spot is defined as the detector
+                cross-section, typically rectangular
+            spot_width - width of the detector
```

Comment:
Same as for spot_shape, worth specifying "detector **element**".

Also, here we have the inconsistency that the pinhole camera detector element is typically rectangular (as you also state) but you don't have it implemented :-/.

Is it worth getting the rectangular spot implemented now?